### ISSUE TITLE
Is the lib still maintained ?

### ISSUE THREAD

**Issue by jobberma** [archriss/react-native-snap-carousel#1031]
Hello,

I am facing some problems with RTL in the lib and I wonder if it's still maintained ??
Thanks for your feedback.

**Comment by andresouza-maple** [comment]
I don't think so, you should try this lib instead https://github.com/dohooo/react-native-reanimated-carousel
